### PR TITLE
GAT-1679

### PR DESCRIPTION
**Pull request by kymmeh**

### CODE REVIEW

File: src/pages/AccountTeamManagementPage/AccountTeamManagementPage.jsx [hunk 1]
```diff
@@ -205,7 +205,8 @@ const AccountTeamManagementPage = ({
     };
 
     // send email notifications to team email address
-    const toggleTeamNotifications = (checked, e = {}, id = '') => {
+    const toggleTeamNotifications = ({ target: { checked, id } }) => {
+        console.log('not toggled');
```
Comment (kymmeh) [inline src/pages/AccountTeamManagementPage/AccountTeamManagementPage.jsx:209]: @ThinkingMonkeyLtd this isn't happening. Any idea why?

File: src/pages/AccountTeamManagementPage/AccountTeamManagementPage.components.jsx [hunk 2]
```diff
@@ -163,13 +163,13 @@ const TeamNotifications = ({
...
 TeamNotifications.propTypes = {
-    teamGatewayNotifications: teamGatewayNotificationsType.isRequired,
+    teamGatewayNotifications: teamGatewayNotificationsPropTypes.isRequired,
     teamId: PropTypes.string.isRequired,
     toggleTeamNotifications: PropTypes.func.isRequired,
     handleRemoveClick: PropTypes.func.isRequired,
     handleFieldChange: PropTypes.func.isRequired,
     handleAddClick: PropTypes.func.isRequired,
-    userState: userStateType.isRequired,
+    userState: userStatePropTypes.isRequired,
```
Comment (ThinkingMonkeyLtd) [inline src/pages/AccountTeamManagementPage/AccountTeamManagementPage.components.jsx:172]: If it's not beyond the scope of this ticket can we strip userState out as useAuth is used at the lower levels. Save all the drilling aswell

File: src/pages/AccountTeamManagementPage/AccountTeamManagementPage.jsx (hunk 1 above)
Comment (ThinkingMonkeyLtd) [reply src/pages/AccountTeamManagementPage/AccountTeamManagementPage.jsx:209]: It looks fine to me. Team mmail is working the same way. Are you sure the right method is being passed down and it's not calling something else?

**Review by ThinkingMonkeyLtd — CHANGES_REQUESTED** [review]

File: src/pages/AccountTeamManagementPage/AccountTeamManagementPage.components.jsx (hunk 2 above)
Comment (kymmeh) [reply src/pages/AccountTeamManagementPage/AccountTeamManagementPage.components.jsx:172]: I'll do that in the next refactor task

**Review by ThinkingMonkeyLtd — APPROVED** [review]

**Pull request merged**